GeoRasterBlueprint/Model/AbstractAnimal.cs

```diff
@@ -69,7 +69,8 @@ public abstract class AbstractAnimal : IPositionable, IAgent<LandscapeLayer> {
             
             var targetPosition = Position.GetRelativePosition(randomDirection, randomDistance);
 
-            if (Perimeter.IsPointInside(targetPosition) && !WaterLayer.IsIntersectsAny(Position, targetPosition)) {
+            // removed for being bugged && !WaterLayer.IsIntersectsAny(Position, targetPosition)
+            if (Perimeter.IsPointInside(targetPosition)) {
```

Comment:
Hier müsste man noch prüfen, dass die neue Position nicht im Wasser ist.

Reply:
Ich habe RandomWalk getestet und es läuft ohne Probleme. Der Bug ist in der MoveToWaterSource() Methode.